templates/commands/goldentest/verify_test.go

```diff
@@ -112,6 +112,30 @@ kind: 'GoldenTest'`
 			},
 			wantErrs: []string{"b.txt] expected, however missing"},
 		},
+		{
+			name: "failed_to_render_test_case",
+			filesContent: map[string]string{
+				"spec.yaml": `apiVersion: 'cli.abcxyz.dev/v1beta5'
+kind: 'Template'
+
+
+inputs:
+  - name: 'my_input_without_default'
+    desc: 'An input without a default'
+
+steps:
+  - desc: 'Print input values'
+    action: 'print'
+    params:
+      message: |
+        The variable values are:
+          my_input_without_default={{.my_input_without_default}}`,
+				"testdata/golden/test/test.yaml": testYaml,
+			},
+			wantErrs: []string{
+				"failed to render test case [test] for template location",
```

Comment:
Perhaps we should check more of the error message. That might have caught the bug where the wrong error variable was printed.

Reply:
I can't check more of the error message is because I won't know the value of template location in advance.

Reply:
Ah, I see, thanks